Makefile

```diff
@@ -48,11 +49,16 @@ test: ## Run tests
 
 # build packages
 
-build: clean ## Build an sdist and wheel
+build: ## Build an sdist and wheel
+	$(python) -m pip install --upgrade setuptools wheel
 	$(python) setup.py sdist bdist_wheel
 .PHONY: sdist
 
+test-release: build
+	./test-release
+.PHONY: test-release
+
 upload: ## Upload our sdist and wheel
-	$(twine) upload dist/*
+	$(twine) upload dist/colorama-$(version)-*
```

Comment:
Independent of this PR's main point, we now just try to upload the current version, not everything in the dist dir.